core/controllers/admin_test.py

```diff
@@ -1467,12 +1475,14 @@ def test_add_question_reviewer(self):
                 'review_category': 'question'
             }, csrf_token=csrf_token)
 
-        self.assertTrue(user_services.can_review_question_suggestions(
```

Comment:
Actually this is interesting -- I have a feeling something is wrong here. All these cases are fine and don't need to be corrected.